tasks/task08/Shape2DCollection.py

```diff
@@ -0,0 +1,24 @@
+from Shape2D import Shape2D, Point2D
+
+class ShapeCollection(Shape2D):
+
+    def __init__(self, stores: list[Shape2D]):
+        self._stores = stores
+
+    def area(self) -> float:
+        sum_area: float = 0
+        for shape in self._stores:
+            sum_area += shape.area()
+        return sum_area
+    
+    def __contains__(self, point: Point2D) -> bool:
+        for shape in self._stores:
+            if shape.__contains__(point):
```

Comment:
`__contains__` is a magic method and should not be called directly (https://stackoverflow.com/questions/1964934/what-does-contains-do-what-can-call-contains-function)